Modules/Loadable/Markups/VTKWidgets/vtkSlicerMarkupsWidgetRepresentation3D.cxx

```diff
@@ -94,8 +94,9 @@ vtkSlicerMarkupsWidgetRepresentation3D::ControlPointsPipeline3D::ControlPointsPi
   this->SelectVisiblePoints = vtkSmartPointer<vtkSelectVisiblePoints>::New();
   this->SelectVisiblePoints->SetInputData(this->LabelControlPointsPolyData);
   this->SelectVisiblePoints->SetTolerance(0.0); // we will set tolerance in world coordinate system
+  this->SelectVisiblePoints->SetOutput(vtkNew<vtkPolyData>());
 
-  this->PointSetToLabelHierarchyFilter->SetInputConnection(this->SelectVisiblePoints->GetOutputPort());
+  this->PointSetToLabelHierarchyFilter->SetInputData(this->SelectVisiblePoints->GetOutput());
```

Comment:
Add a comment here to ensure that nobody will accidentally change the code back to SetInputConnection.

Reply:
Done!